glossary.html

```diff
@@ -15,15 +15,11 @@
     <header id="top">
         <nav>    
             <ul>
-              <div class="box">
```

Comment:
Only `<li>` elements may be the immediate children of `<ul>` elements.  If you added this element in order to allow yourself to use a class selector in the CSS, consider using a descendant selector instead.  Eg.  instead of `.box { ... }` use something like `header nav ul`